test/test_gpt2tok.py

```diff
@@ -4,7 +4,7 @@
 
 from pathlib import Path
 from onnx import helper, onnx_pb as onnx_proto
-from transformers import GPT2Tokenizer
+from transformers import *
```

Comment:
Don't import everything, especially for this large package.

Reply:
Sure, thanks.